### DISCUSSION TITLE
net snmp adding xDSL stats

### DISCUSSION THREAD

**Discussion by francisuk1989** [Freetz-NG/freetz-ng#998] [Unerfüllte Wünsche]
Would it be possible to add some xDSL stats for net-snmp.

**Comment by fda77** [comment]
Sure but you need someone who does it.
I'm using such a non-public script with "cacti". You can see eg a multiple day log interruption and the upgrade from 10 to 20 mbit upload

![image](https://github.com/Freetz-NG/freetz-ng/assets/471850/e9cea2ba-81ef-40a4-8996-f8e19683305a)
